avalon_enforcer/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.mti
+*.mpf
+transcript
+*.wlf
+wlftznre3x
+/work
+wlft1qbgah                                                                                                             
+wlft713cbh                                                                                                              
+wlftfi9h67 
+/ignore
```

Comment:
You could have used just the name of the dir the modelsim project is in. For example if the modelsim files (non-rtl) are in modelsim dir, you can write /modelsim, or /modelsim/*

This works though only if you save the rtl and tb code in a seperate dir from the modelsim project (which should be the case)